styles/styles.css

```diff
@@ -162,16 +163,28 @@ body.no-scroll {
   overflow: hidden;
 }
 
-main .section .default-content-wrapper { /* mobile */
-  padding: 0 var(--body-padding);
-}
-
 header {
   height: var(--nav-height);
 }
 
+.section {
+  width: 100%;
+  position: relative;
+  padding: 32px 0;
```

Comment:
Please set a variable similar to width --section-mobile-padding and desktop padding

Reply:
done